app/controllers/api/v1/push/subscriptions_controller.rb

```diff
@@ -1,26 +1,31 @@
 # frozen_string_literal: true
 
 class Api::V1::Push::SubscriptionsController < Api::BaseController
+  include Redisable
+  include Lockable
+
   before_action -> { doorkeeper_authorize! :push }
   before_action :require_user!
-  before_action :set_push_subscription
+  before_action :set_push_subscription, only: [:show, :update]
```

Comment:
Ignorable style advice: wrap this line and one below it in `with_options only: ...` block